CMakeLists.txt

```diff
@@ -96,10 +96,6 @@ if(ICD_BUILD_LLPC)
 
     add_subdirectory(llpc ${PROJECT_BINARY_DIR}/llpc)
 
-    if(LLPC_BUILD_TESTS OR LLPC_BUILD_LIT)
-        add_subdirectory(test)
-    endif()
-
```

Comment:
Is this an intentional change?

Reply:
no, I had to do that to get the standalone build to work.

Reply:
This is fixed in #1487